apps/client/src/features/viewers/studio/StudioClock.scss

```diff
@@ -81,8 +81,8 @@ $cyan-idle: #0aa;
 
     .studio-timer {
       color: $red-active;
-      font-size: calc(#{$clock-size} / 3);
-      margin-top: calc(50% - calc(#{$clock-size} / 7));
+      font-size: calc(#{$clock-size} / 4.6);
+      margin-top: calc(50% - calc(#{$clock-size} / 11));
```

Comment:
Would it be possible to have these sizes dynamic on whether we show seconds?
The easiest way to achieve it would be to use a CSS class, otherwise, we could look into the useFitText hook (bit fiddly, but very powerful)

Reply:
after fiddeling around with useFitText for about thre hours i've decided to go the CSS class way. You are now able to show the seconds vie the ?seconds=true url parameter